### ISSUE TITLE
quill.js:2037 quill Cannot import modules/sanitize

### ISSUE THREAD

**Issue by Timebutt** [KillerCodeMonkey/ngx-quill#752]
Hi,

I am adding `ngx-quill` to my Angular project running Angular `9.0.0-rc.9` and keep seeing console errors every time either the `quill-editor` or `quill-view` component renders. These errors appear:

```
quill Cannot import modules/sanitize. Are you sure it was registered?
quill Cannot load sanitize module. Are you sure you registered it?
```

This happens when I set `sanitize`  to `false` in `QuillModule.forRoot()`. If I set it to `true` I'm seeing the above errors with this one on top:

```
TypeError: moduleClass is not a constructor
```

I've seen in other issues that `Ivy` works for Angular 8 versions, but I'm guessing something's completely off in Angular 9? The Angular 9 release is right around the corner and many people will be upgrading as it releases I am sure. 

For reference: I run a very basic configuration of `ngx-quill` and everything seems to be working when `sanitize` is set to `false` other than the console errors I'm seeing. I've done the setup per the documentation, and as it's working and rendering fine I don't think an incorrect configuration is the issue at hand.

Have you gotten this library to run with Angular version 9? I'm guessing there might be a breaking change in their API somewhere?

Thanks!

**Comment by KillerCodeMonkey** [comment]
I think you pass an invalid modules config

There is no sanitize module. If you want to sanitize your content you need
to pass the sanitize Input to the components. An example can be found in my
Demo repo.

Nils Tijtgat <notifications@github.com> schrieb am Sa., 1. Feb. 2020, 11:10:

**Comment by Timebutt** [comment]
Thanks for your very quick response ;)

I see, I misinterpreted this from the [documentation](https://github.com/KillerCodeMonkey/ngx-quill#security-hint) then. It says `Ngx-quill provides the config paramter sanitize to sanitize html-strings`. A config parameter to me is something that's set in configuration, what you are referring to is an `Input property` in the example right? 

I think it would definitely clarify things if you linked to the example you posted above, or if the documentation would be rephrased just a bit? I think I got confused cause VSCode offered me `sanitize` as an Intellisense option in the config object, probably getting it from some other module I have in my application ...

Thanks again!
